bazel/external/wee8.genrule_cmd

```diff
@@ -30,16 +42,23 @@ rm -rf out/wee8
 # Export compiler configuration.
 export CXXFLAGS="$${CXXFLAGS-} -Wno-sign-compare -Wno-deprecated-copy -Wno-unknown-warning-option -Wno-range-loop-analysis -Wno-shorten-64-to-32"
 if [[ ( $${SYSTEM} == "Darwin" && $${CXX-} == "" ) || $${CXX-} == *clang* ]]; then
-  export IS_CLANG=true
   export CC=$${CC:-clang}
   export CXX=$${CXX:-clang++}
-  export CXXFLAGS="$${CXXFLAGS} -Wno-implicit-int-float-conversion -Wno-builtin-assume-aligned-alignment -Wno-final-dtor-non-final-class -Wno-invalid-constexpr"
 else
-  export IS_CLANG=false
   export CC=$${CC:-gcc}
   export CXX=$${CXX:-g++}
 fi
 
+case $$(detect::compiler) in
```

Comment:
Wouldn't it be easier to simply inline `if $${CXX} --version 2>&1 | grep -q clang ; then` here?